### ISSUE TITLE
Dungeon Ticket & Quest

### ISSUE THREAD

**Issue by xwerswoodx** [Ishadijcks/pokeclicker#202]
Because of high price of Dungeon Ticket and with impossible events you can't continue game and stuck on route 3.

![asd](https://user-images.githubusercontent.com/6155640/33612869-dcc1953e-d9ca-11e7-9977-ca2217d916d8.png)

This is my quest at route 3.
Let me explain one by one.

Quest 1: Catch 373 pokemon.
To catch pokemon you need to buy pokeballs. I bought 100 pokeballs (and a pokeball price gone up to 700 each) when my pokeballs finished, I had 85 pokemons and 623 price for each pokeball, also nearly earned 4k.  So if you divide 4000 to 623, you can get 6.4 pokeballs which means 6 pokeballs. So at the beginning, it's hard to earn so much money, and this is nearly impossible, or takes so much time to continue. (May take more than 3-4 hours)

Quest 2: Get benefit from Catch Ball.
It's nearly same with Quest 1, because you need pokeballs for it, it slightly easy, but 100 QP don't let you continue, or you are not even close to get 1k for Dungeon Ticket.

Quest 3: Gain 7313 Dungeon Points
As I explained a bug here: https://github.com/Ishadijcks/pokeclicker/issues/201
This is impossible of impossible. Because you need to catch pokemons to earn dungeon coins, and each pokemon gives 3 DC at this route. so you need to caught (7373/3) 2458 pokemons to finish this quest, and you can't catch every pokemon so minimum you need 3.5k pokeballs to finish this quest, as I explained at Quest 1, it's too hard for beginning. (Also you can't use token collector)

Quest 4: Defeat Saffron GYM
Can't go Saffron GYM yet.

Quest 5: Get benefit from amulet
I don't have that item, I already caught all types of pokemon on route 1,2 and 3 and evolved them to most (except pokemons need moon stone) and total pokemon count is 25. (Amulet Coin needs 30 pokemon)

Quest 6: Pokemon Mansion
Can't go Cinnabar Island yet. (Also still need dungeon ticket for it)

Quest 7: 5000+ dungeon coins
Check QUEST 3.

Quest 8: 7000+ dungeon coins
Check QUEST 3.

Quest 9: Amulet Coin Benefit
Check QUEST 5.

Quest 10: Mine 1 Layers
Underground doesn't open yet, you can't mine.

----------------

In short, if you do this https://github.com/Ishadijcks/pokeclicker/issues/201 bug fix, we can make quest about dungeon coins to continue, without this fix, it's slightly impossible to continue without earn so much money and buy maybe more than 10000 pokeballs in one time to prevent yourself by price increase in shop.

Or maybe you can fix this with make quest depends on highest level pokemon on backpack (don't use starter because maybe he give his starter for breed) and gym quest can check if he already went this town, or maybe dungeon ticket could be cheaper, I don't know, but with this quest and price, we can't even continue to play.

**Comment by Aegyo** [comment]
There will be some kind of main questline (defeat Brock, etc) guiding players where to go. These quests will give you enough quest points to afford the ticket before you need it.

**Comment by Aegyo** [comment]
I do think that offering these kind of quests so early in the game isn't such a great idea though (because they are mostly not possible), but I didn't have a great way of preventing them while retaining the "same quests for everyone" thing.

Possible solution would be to lock regular quests behind the main questline I mentioned before, which would be finished at the elite 4. Not sure how that would work with multiple regions however.

**Comment by xwerswoodx** [comment]
Yeah maybe at the beginning, we can do something like beginner quest to earn quest coins to continue

**Comment by Ishadijcks** [comment]
You only need a dungeon ticket once to unlock them. The 'main' questline will give you enough to buy it.

**Comment by Ishadijcks** [comment]
Now explained in #224
